functional/keylime_create_policy-static-data/test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+rlJournalStart
+
+    rlPhaseStartSetup "test1: No input"
```

Comment:
This and all the following `rlPhaseStartSetup` should be `rlPhaseStartTest` instead. There is a difference how Setup and Test phases are processed by Beakerlib when it comes to errors so better to use Test to ensure that errors are propagated properly.

Reply:
Fixed.